src/main/java/cz/it4i/fiji/datastore/security/OAuthUserNew.java

```diff
@@ -0,0 +1,37 @@
+package cz.it4i.fiji.datastore.security;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+import io.quarkus.hibernate.orm.panache.PanacheEntityBase;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Entity
+@Table(name = "oauth_user")
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+public class OAuthUserNew extends PanacheEntityBase {
```

Comment:
Is PanacheEntityRepository necessary? I think, there is used only EntityManager.